File: Entidades/Bitacora.vb
'Continene: Id_Credito, Fecha_Hora, Descripcion
Public Class Bitacora

    'Variable extraída de la tabla CLIENTE_CREDITO y que específica a cual cliente pertenece la bitacora.
    Private _varIdCredito As Integer
    Public Property IdCredito() As Integer
        Get
            Return _varIdCredito
        End Get
        Set(ByVal value As Integer)
            _varIdCredito = value
        End Set
    End Property

    'Fecha y hora en que fue creada la bitacora
    Private _varFechaHora As String
    Public Property FechaHora() As String
        Get
            Return _varFechaHora
        End Get
        Set(ByVal value As String)
            _varFechaHora = value
        End Set
    End Property

    'Descripción o motivo de la creación de la bitacora.
    Private _varDescripcion As String
    Public Property Descripcion() As String
        Get
            Return _varDescripcion
        End Get
        Set(ByVal value As String)
            _varDescripcion = value
        End Set
    End Property
End Class

File: Entities/Bitacora.cs
namespace Entidades
{
    using System;

    public class Bitacora
    {
        private int _varIdCredito;
        private string _varFechaHora;
        private string _varDescripcion;

        public int IdCredito
        {
            get => 
                this._varIdCredito;
            set => 
                this._varIdCredito = value;
        }

        public string FechaHora
        {
            get => 
                this._varFechaHora;
            set => 
                this._varFechaHora = value;
        }

        public string Descripcion
        {
            get => 
                this._varDescripcion;
            set => 
                this._varDescripcion = value;
        }
    }
}

